integration-cli/docker_cli_build_test.go

```diff
@@ -2293,6 +2293,29 @@ func TestBuildExposeOrder(t *testing.T) {
 	logDone("build - expose order")
 }
 
+func TestBuildExposeHostPort(t *testing.T) {
```

Comment:
what I mean is... you added a new warning message into the flow, so your testcases should force the warning to appear and verify that the CLI sees it. So, the Dockerfile should have the EXPOSE and then you should check the output of buildImage() to make sure your new warning message appears.